### ISSUE TITLE
Uncaught Error: spawn EACCES

### ISSUE THREAD

**Issue by laughedelic** [AtomLinter/linter-scalac#20]
**Atom Version**: 1.0.4
**System**: Mac OS X 10.10.5
**Thrown From**: [linter-scalac](https://github.com/AtomLinter/linter-scalac) package, v1.0.1

### Stack Trace

Uncaught Error: spawn EACCES

```
At /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:260

Error: spawn EACCES
  at exports._errnoException (util.js:734:11)
  at ChildProcess.spawn (child_process.js:1145:11)
  at Object.exports.spawn (child_process.js:977:9)
  at BufferedProcess.module.exports.BufferedProcess.spawn (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:188:44)
  at new BufferedProcess (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:52:14)
  at /Users/laughedelic/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:40:30
  at Object.module.exports.Helpers._exec (/Users/laughedelic/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:22:16)
  at Object.module.exports.Helpers.exec (/Users/laughedelic/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:13:13)
  at Object.provider.lint (/Users/laughedelic/.atom/packages/linter-scalac/lib/linter-scalac.coffee:42:24)
  at Promise.then._this.emitter.emit.linter (/Users/laughedelic/.atom/packages/linter/lib/linter-registry.coffee:55:22)
  at LinterRegistry.triggerLinter (/Users/laughedelic/.atom/packages/linter/lib/linter-registry.coffee:54:16)
  at /Users/laughedelic/.atom/packages/linter/lib/linter-registry.coffee:48:17
  at Array.map (native)
  at /Users/laughedelic/.atom/packages/linter/lib/linter-registry.coffee:46:27

```

### Commands

```
     -6:15.9.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -2:43.8.0 pane:show-next-item (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -2:27.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -1:14.7.0 core:cut (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:14.2.0 editor:move-to-first-character-of-line (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:13.7.0 core:paste (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:12.5.0 editor:move-to-beginning-of-word (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:11.9.0 core:move-left (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:11.7.0 core:backspace (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:10.9.0 editor:move-to-end-of-screen-line (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -1:07.9.0 core:cut (atom-text-editor#linter-scalac.scalacOptions.editor.mini.is-focused)
     -0:21.1.0 pane:show-previous-item (div.settings-view.pane-item)
     -0:13.2.0 vim-mode:move-to-next-whole-word (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:12.7.0 vim-mode:move-right (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:12.3.0 vim-mode:delete-right (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:10.7.0 core:save (atom-text-editor.editor.vim-mode.is-focused.normal-mode)
```

**Comment by jager13579** [comment]
In the README, you aren't suppose to include "scalac" in the path. However, it looks like the fix for #19 reverted this. I'll submit a pull request to fix the code and make it match the documentation.

**Comment by Arcanemagus** [comment]
Marking this as closed, this looks like it was a configuration issue with the wrong path to `scalac` being put in.

If you are still having issues @laughedelic please comment here or file a new issue.
